tidy3d/components/simulation.py

```diff
@@ -1797,16 +1798,19 @@ def make_eps_data(coords: Coords):
         return make_eps_data(coords)
 
     @classmethod
-    def from_file(cls, fname: str, **parse_kwargs):
+    def from_file(cls, fname: str, validate=True, **parse_kwargs) -> Simulation:
         """Loads a :class:`Tidy3dBaseModel` from .yaml or .json file.
 
         Parameters
         ----------
         fname : str
             Full path to the .yaml or .json file to load the :class:`Tidy3dBaseModel` from.
+        validate : bool
+            If ``False``, load without calling any validators. NB: should only be used with trusted
```

Comment:
out of curiosity, what is "NB"? google search suggests "nota bene"?

Reply:
Yeah it stands for "nota bene". Sorry, this is probably not as common as it is in my head... I will just replace with "Warning: "